packages/expo-barcode-scanner/ios/EXBarCodeScanner/EXBarCodeScannerProvider.m

```diff
@@ -2,6 +2,7 @@
 
 #import <EXBarCodeScanner/EXBarCodeScannerProvider.h>
 #import <EXBarCodeScanner/EXBarCodeScanner.h>
+#import <ExpoModulesCore/EXDefines.h>
```

Comment:
was missing and the module was not registered - led to some import failures

Reply:
cc @tsapeta

Reply:
Yeah, I think that's correct change since `EX_REGISTER_MODULE` macro that we use here is defined in `EXDefines.h`